src/I18nBundle/Manager/ZoneManager.php

```diff
@@ -488,7 +488,7 @@ private function mapDomainData($domain, $rootId)
             'countryIso'       => $docCountryIso,
             'languageIso'      => $docRealLanguageIso,
             'hrefLang'         => $hrefLang,
-            'localeUrlMapping' => null,
+            'localeUrlMapping' => (!empty($docLocale)) ? $docLocale : null,
```

Comment:
@Lukaschel I think it needs to be `count($subPages) === 0 && !empty($docLocale) ? $docLocale : null`. If there are valid localized subpages, the root page is not allowed to have any locale mapping.

## example a
mydomain.com
 - de
 - en

=> localeUrlMapping for mydomain.com needs to be empty

## example b
mydomain.com
=> localeUrlMapping for mydomain.com not need to be empty (thats also your usecase, right?)

Reply:
yes you are right it also need the subpages checkup